tap_square/client.py

```diff
@@ -96,6 +97,7 @@ def get_employees(self, bookmarked_cursor):
             yield (result.body.get('employees', []), result.body.get('cursor'))
 
     def get_locations(self):
+        body = {}
```

Comment:
What is the purpose of storing this as a variable that you keep after the loop where it's used later?